superset-frontend/src/components/ProgressBar/ProgressBar.stories.tsx

```diff
@@ -24,37 +25,67 @@ export default {
 };
 
 export const InteractiveProgressBar = (args: ProgressBarProps) => (
-  <ProgressBar {...args} />
+  <AntdThemeProvider>
+    <ProgressBar {...args} type="line" />
+  </AntdThemeProvider>
 );
 
-InteractiveProgressBar.args = {
+export const InteractiveProgressCircle = (args: ProgressBarProps) => (
+  <AntdThemeProvider>
```

Comment:
@geido Can't we include the `AntdThemeProvider` in the [preview.jsx](https://github.com/apache/superset/blob/master/superset-frontend/.storybook/preview.jsx) file?

Reply:
We have tried but got not much luck with that yet. We'll continue trying.